reV/supply_curve/supply_curve.py

```diff
@@ -244,7 +253,11 @@ def _merge_sc_trans_tables(cls, sc_points, trans_table,
             Table of supply curve point summary
         trans_table : pd.DataFrame | str
             Table mapping supply curve points to transmission features
-            (either str filepath to table file or pre-loaded dataframe).
+            (either str filepath to table file, list of filepaths to tables by
+             line voltage (capacity) or pre-loaded dataframe).
+        compare_cap : bool, optional
+            Flag to compare the sc_point and transmission feature capacities,
+            'capacity' <= 'max_cap'
```

Comment:
So this came up during the recursion. Originally I was going to drop the min/max_cap columns from the trans_tables but figured it would be good to keep those for debugging. The flag isn't exposed to the user its only used inside the method during the recursion. Its on by default but turned off here:  https://github.com/NREL/reV/blob/c42b431184d6f8ce9ea3bcb50472d9cbe45d3955/reV/supply_curve/supply_curve.py#L289

when we are iterating through the different capacity "bins" This is to make sure we don't end up in a recursion loop

Reply:
hmmm okay. the reason for this still kind of confuses me. the goal here is to join transmission tables with SC agg tables for different max capacities? Could we also do a left join and drop duplicates based on the capacity comparison? That would reduce one confusing level of recursion and also make a natural location to check for duplicates.

Reply:
I don't follow how that would work. The inputs are as follows:
sc_gids of interest
each trans_table contains all sc_points and their connection to the transmission features in the given bin (> min_cap, <= max_cap)

We want to pull the transmission feature from the proper table for each sc_gid based on the sc_gids capacity. I don't follow how we can properly delete duplicates. Do we just join the tables and then delete duplicate trans_gids based on the transmission capacity (retaining the smallest capacity feature?)